conf.py

```diff
@@ -30,10 +30,10 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
 # setting title in left panel (default is <projectname> documentation <release>)
-#html_title = "CSC Geocomputing course"
+#html_title = "Geocomputing course"
```

Comment:
```suggestion
#html_title = "Geocomputing with Puhti supercomputer course"
```

Reply:
(this one is not used, it would be used as text if no html image were given.